database/things/contracts.coffee.pldb

```diff
@@ -0,0 +1,15 @@
+title contracts.coffee
```

Comment:
this is an interesting one @lngns ! normally I would have named this file "contracts-dot-coffee.pldb", but I cannot explain why we can't just use the dot. Good call. I am going to change the other files to follow your pattern.